ntoskrnl/kd64/kdprint.c

```diff
@@ -343,31 +343,16 @@ KdpPrint(
     STRING OutputString;
     CHAR CapturedString[512];
 
-    /* Assume failure */
-    *Handled = FALSE;
-
-#if (NTDDI_VERSION >= NTDDI_VISTA)
-    if ((ComponentId >= KdComponentTableSize) && (ComponentId < MAXULONG))
-    {
-        /* Use the default component ID */
-        Mask = &Kd_DEFAULT_Mask;
-        // Level = DPFLTR_INFO_LEVEL; // Override the Level.
-    }
-#endif
-    /* Convert Level to bit field if required */
-    if (Level < 32) Level = 1 << Level;
-    Level &= ~DPFLTR_MASK;
```

Comment:
Could you (in your local copy) check whether just removing that `Level &= ~DPFLTR_MASK;` line somewhat "improves" things?

Reply:
We have this check in both functions so it should not change anything (and it is not, I've checked :))